### PR TITLE
[Server Maintenance] migrate from shoryuken to sidekiq

### PR DESCRIPTION
**Pull request by rusllonrails**
[Trello Story](https://trello.com/c/Mt718lLX/281-qae-support-server-maintenance)

### CODE REVIEW

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+redis_url = ENV.fetch("REDIS_URL", "redis://localhost:6379")
+namespace = "qae_#{Rails.env}"
+
+Sidekiq.configure_server do |config|
+  config.redis = {
+    url: redis_url,
+    namespace: namespace
```
Comment (matthewford) [inline config/initializers/sidekiq.rb]: https://github.com/mperham/sidekiq/blob/master/4.0-Upgrade.md

> redis-namespace has been removed from Sidekiq's gem dependencies.

We should not be using it.

File: Gemfile [hunk 2]
```diff
@@ -80,7 +80,8 @@ gem "fog-aws"
 gem "vs_rails", github: "bitzesty/vs-rails"
 
 # Background jobs
-gem "shoryuken", github: "phstc/shoryuken", branch: "master"
+gem "sidekiq"
+gem "redis-namespace"
```
Comment (matthewford) [inline Gemfile]: remove

File: Gemfile [hunk 3]
```diff
@@ -80,7 +80,8 @@ gem "fog-aws"
 gem "vs_rails", github: "bitzesty/vs-rails"
 
 # Background jobs
-gem "shoryuken", github: "phstc/shoryuken", branch: "master"
+gem "sidekiq"
```
Comment (matthewford) [inline Gemfile]: we should be more explicit in the versions in the Gemfile going forward to make updating one thing at a time simpler.

Comment (matthewford) [inline Gemfile]: e.g ~>"4.1.0"

**Comment by matthewford** [comment]
otherwise looks good

**Comment by macool** [comment]
merge after #753

**Comment by rusllonrails** [comment]
:+1:

**Pull request merged**